src/model/test/ChillerElectricEIR_GTest.cpp

```diff
@@ -657,3 +660,48 @@ TEST_F(ModelFixture, ChillerElectricEIR_PlantLoopConnections_addToNodeOverride)
   ASSERT_TRUE(chiller.heatRecoveryLoop());
   EXPECT_EQ(hrLoop, chiller.heatRecoveryLoop().get());
 }
+
+TEST_F(ModelFixture, ChillerElectricEIR_ElectricInputToCoolingOutputRatioFunctionOfPLR) {
+  // Test for #4611 - Allow non-Quadratic curves for the EIR-f-PLR for the Chiller:Electric:EIR object
+
+  model::Model model;
+
+  // test ctor 1
+  {
+    // ctor maintains backward compatibility
+    model::ChillerElectricEIR chiller(model);
+    ASSERT_TRUE(chiller.electricInputToCoolingOutputRatioFunctionOfPLR().optionalCast<model::CurveQuadratic>());
+  }
+
+  // test ctor 2
+  {
+    // ctor maintains backward compatibility
+    model::CurveBiquadratic ccFofT(model);
+    model::CurveBiquadratic eirToCorfOfT(model);
+    model::CurveQuadratic eirToCorfOfPlr(model);
+    model::ChillerElectricEIR chiller2(model, ccFofT, eirToCorfOfT, eirToCorfOfPlr);
+    ASSERT_TRUE(chiller2.electricInputToCoolingOutputRatioFunctionOfPLR().optionalCast<model::CurveQuadratic>());
```

Comment:
I'm going to make a small modification to test curve equality (might as well test more than less)